R/mail.R

```diff
@@ -0,0 +1,54 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox data
+#'
+#' @param mlist specific mailing list associated with this mbox data
+#' @param reply_from email of sender
+#' @param reply_to email of receiver
+#' @param reply_cc carbon copy of email to additional recipients
+#' @param reply_datetime Date, time in the following format: 2023-02-11T09:30:00
```

Comment:
where's the timezone here? I waived the unit test for it, but we still need to capture it.